qCC/ccEntityAction.cpp

```diff
@@ -1107,11 +1107,39 @@ namespace ccEntityAction
 				
 				CCCoreLib::ScalarField* sf = pc->getScalarField(sfIdx);
 				Q_ASSERT(sf->currentSize() == pc->size());
-				
-				for (unsigned j=0 ; j<cloud->size(); j++)
+
+				for (size_t j=0 ; j<cloud->size(); j++)
 				{
-					ScalarType idValue = static_cast<ScalarType>(j);
-					sf->setValue(j, idValue);
+					ScalarType idValue;
+					if (false)
+					{
+						idValue = static_cast<ScalarType>(j);
+						sf->setValue(j, idValue);
```

Comment:
I guess you can call this only once (after the if then else statement)

Reply:
Yes